### PR TITLE
Add DR workflows for db backup and restore

### PR DESCRIPTION
**Pull request by RMcVelia**
## Context

Add Postgres database DR workflows for backup and restore

https://trello.com/c/JCJV79eo/2113-add-dr-workflows-to-aftt

## Changes proposed in this pull request

Update existing backup workflow to new template
Add restore workflows from template

## Guidance to review

qa backup: https://github.com/DFE-Digital/apply-for-teacher-training/actions/runs/11614444696
restore workflows will be tested after merge.

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [ ] Inform data insights team due to database changes
- [ ] Make sure all information from the Trello card is in here
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Add PR link to Trello card

### CODE REVIEW

**Comment by RMcVelia** [comment]
[add DR workflows to aftt](https://trello.com/c/JCJV79eo/2113-add-dr-workflows-to-aftt)

File: .github/workflows/postgres-ptr.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Restore database from point in time to new database server
```
Comment (dcyoung-dev) [inline .github/workflows/postgres-ptr.yml:1]: I think I understand this but want to check - if/when we run this does it automatically restore over the top of the current "production" server or does this spin up a separate server?

Comment (RMcVelia) [reply .github/workflows/postgres-ptr.yml:1]: PTR restore spins up a separate server and restores into it.
It doesn't alter the current production server in any way.

**Review by vipin-dfe — APPROVED** [review]
looks good.

**Pull request merged**